src/main/java/com/almightyqa/TestUtils.java

```diff
@@ -0,0 +1,122 @@
+package com.almightyqa;
+
+import com.github.javafaker.Faker;
+import com.google.gson.Gson;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.stream.IntStream;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class TestUtils {
+
+    public static final Random RANDOM = new Random();
+    public static final Faker FAKER = new Faker();
+
+    public static final Gson GSON = new Gson();
+
+    public static int getRandomInt(int... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextInt(bound[0]);
+    }
+
+    public static long getRandomLong(long... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextLong(bound[0]);
+    }
+
+    public static long getRandomNumberOfGivenSize(int numberOfDigits) {
+        long min = 1;
+        if(numberOfDigits>19){
+            throw new IllegalArgumentException("Max number for long is 19 digits long");
+        }
+        for (int i = 1; i < numberOfDigits; i++) {
+            min *= 10;
+        }
+        return min + getRandomLong(9L * min) - 1;
+    }
+
+    public static String getRandomEmail(final String domain) {
```

Comment:
May add getRandomEmail with no argument

Reply:
not GET but generate, this applies to all

Reply:
Without domain it can be confusing and threaten the uniqueness

Reply:
it can be said about many others
and without the argument it will be more used
Also, as in other comment mention, change get to generate